docs/_docs/guides/001-getting-started.md

```diff
@@ -17,7 +17,9 @@ For this guide, we use [Brew](http://brew.sh/) to install {{ site.mojito_green }
 
     mojito-webapp
 
-This starts the server on [http://localhost:8080](http://localhost:8080).  You can log in with `admin/ChangeMe`.
+This starts the server on [http://localhost:8080](http://localhost:8080).  
```

Comment:
Nit: If you aren't giving the link a name, there's a shorthand you can use `<http://localhost:8080>`